base/applications/cmdutils/timeout/lang/ru-RU.rc

```diff
@@ -0,0 +1,33 @@
+LANGUAGE LANG_RUSSIAN, SUBLANG_DEFAULT
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "ReactOS Timeout Utility\n\
+\n\
+TIMEOUT [/?] [/T] таймаут [/NOBREAK]\n\
+\n\
+Описание:\n\
+    Команда позволяет организовать режим ожидания фиксированного интервала времени,\n\
+    в секундах, либо интервала до нажатия клавиши на клавиатуре.\n\
+\n\
+Синтаксис:\n\
+    /?        Показать эту справку.\n\
+\n\
+    /T delay  Время ожидания в секундах. Допустимый интервал: от -1 до 99999 секунд.\n\
```

Comment:
... you should also add it here instead of "delay". (but since "таймаут" writes longer than "delay", you would also need to fix a bit the explanation text alignment to make it still viewable in 80x25 console.